cypress/integration/channel/add_users_to_channel_spec.js

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 6]*/
+
+const channelDisplayNameTest1 = 'Channel test 1';
+const channelDisplayNameTest2 = 'Channel test 2';
+let channelTest1Id;
+let channelTest2Id;
+
+function deleteCurrentChannel() {
+    cy.getCurrentChannelId().then((channelId) => {
+        cy.apiDeleteChannel(channelId);
+    });
+}
+
+function verifyMentionedUserAndProfilePopover(postId) {
+    cy.get(`#post_${postId}`).find('.mention-link').each(($el) => {
+        // # Click each username link
+        cy.wrap($el).click();
+
+        // * The profile pop-over of each user should be displayed when their username is clicked, and the pop-over is fully visible
+        const userName = $el[0].innerHTML;
+        cy.get('#user-profile-popover').should('be.visible');
+
+        // * The pop-over title  at the top of the pop-over should be the same as the username link for each user
+        const escapedUsername = userName.replace('@', '').replace('.', '-');
+        cy.get(`#userPopoverUserName-${escapedUsername}`).should('contain', userName);
+    });
+}
+
+function addUsersToChannel(amount) {
+    for (let i = 0; i < amount; i++) {
+        cy.get('#multiSelectList').first().click();
+    }
+}
+
+describe('Add user to a channel', () => {
+    afterEach(() => {
+        deleteCurrentChannel();
+    });
+
+    it('Single User: Usernames are links, open profile popovers', () => {
+        // # Login as 'user-1', go to / and create 'Channel test 1'
+        cy.apiLogin('user-1');
+        cy.visit('/');
+        cy.getCurrentTeamId().then((teamId) => {
+            cy.apiCreateChannel(teamId, 'channel-test-1', channelDisplayNameTest1).then((res) => {
+                channelTest1Id = res.body.name;
+
+                // # Select 'Channel test 1' channel on the left hand side
+                cy.get(`#sidebarItem_${channelTest1Id}`).click();
```

Comment:
This is unable to click if there are too many channels in the sidebar.  So I recommend,
```suggestion
                cy.get(`#sidebarItem_${channelTest1Id}`).scrollIntoView().click({force: true});
```